requirements.txt

```diff
@@ -1,2 +1,3 @@
 ops ~= 2.4
-requests
\ No newline at end of file
+requests
+pyyaml
```

Comment:
I wonder how it was able to run before without that.

Reply:
`ops` depends on pyyaml. But since we have some charms libs that explicitely depend on it, we should explicitely list it as a dep too.